web/frontend/src/app/component/subset-details/subset-details.component.html

```diff
@@ -118,19 +118,19 @@
                [lazy]="true" (onLazyLoad)="onLazyLoadData($event)" class="table table-striped table-bordered">
         <ng-template pTemplate="header">
           <tr>
-            <th class="w-10">Code</th>
-            <th class="w-10">Source</th>
-            <th class="w-10">Name</th>
-            <th class="w-20">Synonyms</th>
-            <th class="w-20">Source Definitions</th>
-            <th class="w-30">Definitions</th>
+            <th class="w-10">NCIt Concept Code</th>
+            <th class="w-10">{{subsetFormat }} Name</th>
+            <th class="w-10">NCIt Preferred Term</th>
+            <th class="w-20">NCIt Synonyms</th>
+            <th class="w-20">{{subsetFormat}} Definition</th>
```

Comment:
I'm thinking that {{subsetFormat}} is going to be like "NCI", "CTRP", or "Other".  Make sure this is putting the value of the contributing source property (e.g. EDQM-H.. for the example concept).  

OK, looking further down, this may be good, but make sure for the "NCIt Definition" column, you're only showing definitions that belong to source="NCI" (same logic as the format above)